tap_mysql/sync_strategies/binlog.py

```diff
@@ -403,61 +438,68 @@ def _run_binlog_sync(mysql_conn, reader, binlog_streams_map, state, config: Dict
             columns = streams_map_entry.get('desired_columns')
 
             if not catalog_entry:
-                events_skipped = events_skipped + 1
+                events_skipped += 1
 
                 if events_skipped % UPDATE_BOOKMARK_PERIOD == 0:
                     LOGGER.debug("Skipped %s events so far as they were not for selected tables; %s rows extracted",
                                  events_skipped,
                                  rows_saved)
-
             else:
-
                 # Compare event's columns to the schema properties
-                # if a column no longer exists, the event will have something like __dropped_col_XY__
-                # to refer to this column, we don't want these columns to be included in the difference
-                diff = set(filter(lambda k: False if re.match(r'__dropped_col_\d+__', k) else True,
-                                  get_db_column_types(binlog_event).keys())). \
-                    difference(catalog_entry.schema.properties.keys())
+                diff = __get_diff_in_columns_list(binlog_event,
```

Comment:
is this the same functionality but moved into a new function?

Reply:
Yep